package.json

```diff
@@ -13,17 +14,18 @@
     "@viem/anvil": "^0.0.6",
     "@vitest/coverage-v8": "^0.34.2",
     "@wagmi/cli": "^1.3.0",
+    "dprint": "^0.40.2",
     "ethers": "^5.7.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "rome": "^12.1.2",
     "typescript": "^5.0.4",
     "vitest": "^0.34.1",
     "wagmi": "^1.3.11"
   },
   "scripts": {
-    "format": "rome format . --write",
-    "lint": "rome check --apply-unsafe .",
+    "format": "dprint fmt",
+    "format:check": "dprint check",
+    "lint": "biome check .",
```

Comment:
inconsistent usage. Formatter fixes by default and has a seperate check script. Linter checks by default and has a seperate fix script